crnk-core/src/main/java/io/crnk/core/boot/CrnkBoot.java

```diff
@@ -417,10 +439,13 @@ public void setServiceDiscovery(ServiceDiscovery serviceDiscovery) {
 	 * NOTE: This using this feature requires a {@link QuerySpecDeserializer} and it does not work with the
 	 * deprecated {@link QueryParamsBuilder}.
 	 */
+	@Deprecated
```

Comment:
I know a number of application making heacy use of this method rather than the properties (or rather CrnkFeature.set...()). . At some point in would be good to align those to mechanisms (like in Spring with the Properties that can be injected into objects). But till that point it cannot be deprecated.